Gordon360/Services/MembershipService.cs

```diff
@@ -47,7 +47,8 @@ public MembershipService(CCTContext context, IAccountService accountService, IAc
         )
         {
             IQueryable<MembershipView> memberships = _context.MembershipView;
-            if (username is not null) memberships = memberships.Where(m => EF.Functions.Like(m.Username, username));
+            if (username is not null) memberships = memberships.Where(m => m.Username == username);
```

Comment:
@antoniavonto this might need to be changed. `EF.Functions.Like` should work, however, while testing this was the bug point. Either way, you need to consider the cases where AD_Usernames are stored as `Antonia.Vontobel` vs `antonia.vontobel`